sdks/python/container/piputil.go

```diff
@@ -28,7 +28,19 @@ import (
 
 	"github.com/apache/beam/sdks/v2/go/pkg/beam/util/execx"
 )
-
+// It will return the python interpreter type i.e. python or python3 if available else return empty string.
+func pythonVersion() string {
```

Comment:
Let's not duplicate the implementation. Can we find a home for the helper under https://github.com/apache/beam/tree/master/sdks/go/pkg/beam/util/ ?

Reply:
Actually, looks like there is one already: https://github.com/apache/beam/blob/1446295d66d394114d4f3e99980a598b9608881c/sdks/go/pkg/beam/core/runtime/xlangx/expansionx/download.go#L385